src/test/java/teammates/test/cases/browsertests/InstructorFeedbackSessionsPageUiTest.java

```diff
@@ -960,10 +960,12 @@ private void testValidationReload() throws Exception {
         newSession.setFeedbackSessionName("");
         newSession.setEndTime(Const.TIME_REPRESENTS_LATER);
         feedbackPage.clickEditUncommonSettingsButtons();
-        feedbackPage.clickNeverPublishTimeButton();
+        feedbackPage.clickCustomPublishTimeButton();
```

Comment:
Any problems setting this to manual instead?

Reply:
The test tests the functionality where the advance settings will still be visible even if the input is rejected. But the advance option is shown only when any of the options has a non-default value.
https://github.com/TEAMMATES/teammates/blob/49386feafec7ca2cb6fdeca32ad1112d4fda962f/src/main/webapp/dev/js/common/instructorFeedbacks.js#L51-L54

`later` is the default value for response visible time. That is why using manual here will cause this line to fail.
`assertTrue(feedbackPage.isVisible(By.id("responsesVisibleFromColumn")));`

https://github.com/TEAMMATES/teammates/blob/49386feafec7ca2cb6fdeca32ad1112d4fda962f/src/main/webapp/dev/js/common/instructorFeedbacks.js#L24-L27